app/src/main/java/org/fossasia/openevent/dbutils/DbSingleton.java

```diff
@@ -108,23 +108,23 @@ public Session getSessionById (int id) {
 
         cur.moveToFirst();
         while (cur.moveToNext()) {
-            s = new Speaker(
-                    cur.getInt(cur.getColumnIndex(DbContract.Speakers.ID)),
-                    cur.getString(cur.getColumnIndex(DbContract.Speakers.NAME)),
-                    cur.getString(cur.getColumnIndex(DbContract.Speakers.PHOTO)),
-                    cur.getString(cur.getColumnIndex(DbContract.Speakers.BIO)),
-                    cur.getString(cur.getColumnIndex(DbContract.Speakers.EMAIL)),
-                    cur.getString(cur.getColumnIndex(DbContract.Speakers.WEB)),
-                    cur.getString(cur.getColumnIndex(DbContract.Speakers.TWITTER)),
-                    cur.getString(cur.getColumnIndex(DbContract.Speakers.FACEBOOK)),
-                    cur.getString(cur.getColumnIndex(DbContract.Speakers.GITHUB)),
-                    cur.getString(cur.getColumnIndex(DbContract.Speakers.LINKEDIN)),
-                    cur.getString(cur.getColumnIndex(DbContract.Speakers.ORGANISATION)),
-                    cur.getString(cur.getColumnIndex(DbContract.Speakers.POSITION)),
-                    cur.getString(cur.getColumnIndex(DbContract.Speakers.COUNTRY)),
-                    cur.getString(cur.getColumnIndex(DbContract.Speakers.SESSIONS))
-                    );
-            speakers.add(s);
+//            s = new Speaker(
```

Comment:
Same thing, what is up here? You can put this work on a different branch if it is still in progress.